test/functional/wallet_send.py

```diff
@@ -142,7 +142,12 @@ def test_send(self, from_wallet, to_wallet=None, amount=None, data=None,
             return
 
         if locktime:
+            assert_equal(from_wallet.decoderawtransaction(from_wallet.gettransaction(res["txid"])["hex"])["locktime"], locktime)
```

Comment:
In 5101f054ce435ac653a3f5e780fca066e2f25088 "test: test sendall and send do anti-fee-sniping"

nit: `gettransaction` has a `verbose` argument that does the decoding so this doesn't need to call `decoderawtransaction`. Same below.

Reply:
Done